### PR TITLE
Change options from V|Verbose to V|verbose

### PR DESCRIPTION
**Pull request by cxhong**
for issue #5407 

````
[root@boston02 ~]# makentp --Verbose
 Usage: makentp [-h|--help|-v|--version]
       makentp [-a|--all] [-V|--verbose]
[root@boston02 ~]# makentp --verbose
[boston02]: configuring management node: boston02.pok.stglabs.ibm.com.
[boston02]: Will configure chronyd instead.
[boston02]: Calling ... /install/postscripts/setupntp 10.0.0.101
[boston02]: Daemon chronyd configured.
````

### CODE REVIEW

**Comment by gurevichmark** [comment]
There are still some man pages that show `--Verbose`:
```
[gurevich@c910loginx01 xcat-core]$ git grep "\-\-Verbose"
xCAT-client/pods/man1/chzone.1.pod:=item B<-V>|B<--Verbose>
xCAT-client/pods/man1/mkzone.1.pod:=item B<-V>|B<--Verbose>
xCAT-client/pods/man1/rmzone.1.pod:=item B<-V>|B<--Verbose>
xCAT-client/pods/man1/sinv.1.pod:=item B<-V>|B<--Verbose>
[gurevich@c910loginx01 xcat-core]$
```

**Comment by cxhong** [comment]
thanks @gurevichmark ,  convert pod to rst files, seems every line modified.

File: docs/source/guides/admin-guides/references/man1/chzone.1.rst [hunk 1]
```diff
@@ -7,15 +7,15 @@ chzone.1
 
 
 ************
-\ **NAME**\
+\ **NAME**\
```
Comment (neo954) [inline docs/source/guides/admin-guides/references/man1/chzone.1.rst]: It seems these trailing spaces are coming from `pod2rst`.

Comment (neo954) [reply docs/source/guides/admin-guides/references/man1/chzone.1.rst]: We may leave these trailing spaces as is.

**Comment by zet809** [comment]
hi, @gurevichmark and @cxhong , the end line space in rst files are manually removed in PR https://github.com/xcat2/xcat-core/pull/5429, but it seems `pod2rst` will add the space back.
So,  I think it is OK for the modification of rst files.

**Comment by zet809** [comment]
hi, @gurevichmark and @cxhong , the

**Comment by gurevichmark** [comment]
@zet809 Yes. In fact we should not be modifying `.rst` files for man pages. (like #5429 did) `.pod` file should be the only source.

**Comment by zet809** [comment]
The PR is OK, so let me merge it.

**Pull request merged**